test/ctia/task/migration/migrate_es_stores_test.clj

```diff
@@ -158,6 +158,27 @@
     (testing "properly configured migration"
       (is (sut/check-migration-params migration-params)))))
 
+(deftest get-migration-params-test
+  (with-redefs [p/global-properties-atom
+                (let [migration-props {:buffer-size 3,
+                                       :batch-size 100,
+                                       :migration-id "migration-test",
+                                       :restart? false,
+                                       :store-keys "malware,  tool,sighting  ",
+                                       :migrations "identity",
+                                       :confirm? false,
+                                       :prefix "v1.2.0"}]
+                  (fn [] (atom {:ctia {:migration migration-props}})))]
+    (is (= (sut/get-migration-params)
+           {:buffer-size 3
```

Comment:
One has to squint a bit at the code to figure out the diff between `migration-props` and the SUT output.

You could make the diff more explicity with dissoc (for comparing the parts that are `=`) and select-keys (for comparing the parts that are `not=`)